packages/experiments/src/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+
+export interface IPagination {}
+
+export interface IPaginationProps {
+  /**
+   * Required: Selected index
+   */
+  selectedPageIndex: number;
+
+  /**
+   * label for the previous page button
+   */
+  previousLabel: string;
+
+  /**
+   * Required: label for the next page button
+   */
+  nextLabel: string;
+
+  /**
+   * Required: The total number of pages.
+   */
+  pageCount: number;
```

Comment:
I wonder if there is a way to make this optional. It almost feels like this component should have something similar to Pivot component in OUFR where it wraps the items it renders when selecting a tab this way you can internally have the `pageCount` as `this.props.children`.

Reply:
I don't think my idea with wrapping the children would work in this case. I guess one required prop to make this control render would make sense. Thanks for addressing the other questions so quickly.

Reply:
My alias is yudon. We can talk in person if you would like :) 

---
In reply to: [222201041](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222201041) [](ancestors = 222201041)